### PR TITLE
Stimulus page, about us, footer links, and breadcrumbs

### PR DESCRIPTION
**Pull request by loumoore**
A few related things for public pages: new stimulus page, updates to about us page, new links in footer, and breadcrumbs

This is ready for code review, but I think some of the copy on the stimulus page needs review/confirmation -- particularly the links to banking options, hence the WIP.

### CODE REVIEW

**Comment by bengolder** [comment]
@loumoore just converted this to a draft, which is a new useful way to say that this PR is not ready for merging.

When you have a moment, it would be helpful to add screenshots of the finished pages to this PR or to the Pivotal Tracker story.

**Comment by loumoore** [comment]
@bengolder will do. got final copy so i will update it, add screenshots/video, and move this into ready for review when it's all ready.

**Comment by loumoore** [comment]
Note: there were two a11y warnings from the wave plugin that i think are acceptable but wanted to call out: 

1) it doesn't like the breadcrumb linking to Home when it comes right after the GYR logo which also links to home. I'm not sure how that could be resolved. 

2) in one of the Stimulus page FAQ questions we link to a pdf from the IRS, which are not great for accessibility, but i don't think there's an alternative resource here

**Comment by loumoore** [comment]
Screen Shot 2020-05-07 at 2 53 07 PM
Screen Shot 2020-05-07 at 2 52 36 PM
Screen Shot 2020-05-07 at 2 48 05 PM
Screen Shot 2020-05-07 at 2 47 57 PM

**Comment by loumoore** [comment]
not sure what's up with this deploy_to_aptible job failing.

`#!/bin/bash -eo pipefail
git push --tags git@beta.aptible.com:vita-min-staging/vita-min-staging.git $CIRCLE_SHA1:master
Warning: Permanently added the ECDSA host key for IP address '52.21.95.179' to the list of known hosts.

To beta.aptible.com:vita-min-staging/vita-min-staging.git
 ! [rejected]        cf1d0e5f2a8331d4b77f94a56bdf17b5e84c24e8 -> master (non-fast-forward)
error: failed to push some refs to 'git@beta.aptible.com:vita-min-staging/vita-min-staging.git'
hint: Updates were rejected because a pushed branch tip is behind its remote
hint: counterpart. Check out this branch and integrate the remote changes
hint: (e.g. 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Exited with code exit status 1`

**Comment by loumoore** [comment]
the failed check is because of some investigation that was going on with staging when i pushed the last commit. it should be good to go.

File: app/assets/javascripts/ajax_mixpanel_events.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ var ajaxMixpanelEvents = (function () {
         var attributePrefix = "trackAttribute";
         if (key.indexOf(attributePrefix) === 0) {
           var propertyKey = key.substring(attributePrefix.length).toLowerCase();
-          eventData.append("event["+propertyKey+"]", value);
+          eventData.append("event[data]["+propertyKey+"]", value);
```
Comment (loumoore) [inline app/assets/javascripts/ajax_mixpanel_events.js:17]: there are a couple specific properties that are pulled directly out of the event dictionary but new custom properties like these will only be attached if they are in the data dictionary under it.

Comment (jenny-heath) [reply app/assets/javascripts/ajax_mixpanel_events.js:17]: @loumoore  i'm still not totally clear on what effect this has - does it fix the tracking we're currently doing with `data-track-click`? is it related to this PR or just something you noticed?

Comment (loumoore) [reply app/assets/javascripts/ajax_mixpanel_events.js:17]: both :) I wanted to add click tracking to the links on the stimulus page so we know how many people we are sending to the IRS websites especially, and in testing those I found that the data-track-attribute-* elements on existing click tracking were not being sent to mixpanel. So it's a bug fix I stumbled upon in the course of building the stimulus page.

**Pull request merged**